### ISSUE TITLE
Completed builds not posting results to the issue tracker

### ISSUE THREAD

**Issue by jimhester** [Bioconductor/packagebuilder#15]
The build http://staging.bioconductor.org:8000/job/384/ completed (with errors) on all three build machines, but no comment was added to the issue tracker https://tracker.bioconductor.org/issue1336.

Perhaps due to the changes in https://github.com/Bioconductor/packagebuilder/pull/10?

**Comment by b-long** [comment]
cc/ @jimhester @mtmorgan @vobencha @hpages @priscian 

**_TL;DR**_
Everything has been restarted.  We should watch for curl timeouts on Morelia.

_If you're interested..._
In re-running [this job](http://staging.bioconductor.org:8000/job/393/#morelia_buildsrc_anchor) to see if the SPB is fixed, it looked like Morelia was up to it's old ways :

![screenshot from 2015-11-25 12 37 22](https://cloud.githubusercontent.com/assets/66993/11404881/6225cf2a-9372-11e5-8c1f-e8a8bd2c5b0d.png)

I don't think it's necessary to read, but here's the rest of [the context](http://staging.bioconductor.org:8000/job/393/#morelia_buildsrc_anchor).  At first, I was under the impression this was fixed by this PR: https://github.com/Bioconductor/packagebuilder/pull/14 , but in hindsight this looks like it might be something new related to a curl timeout. 

The relevant log files for this job on Morelia, in `/Users/pkgbuild/packagebuilder/jobs/multiClust_20151125115018`

installDeps.log:

```
Loading required package: BiocInstaller
Bioconductor version 3.3 (BiocInstaller 1.21.2), ?biocLite for help
[1] "graph"
[1] "biocViews"
[1] "knitr"
[1] "knitrBootstrap"
[1] "devtools"
[1] "codetools"
[1] "httr"
[1] "curl"

Attaching package: 'curl'

The following object is masked from 'package:httr':

    handle_reset

Downloading GitHub repo Bioconductor/BiocCheck@master
Error in curl::curl_fetch_memory(url, handle = handle) : 
  Timeout was reached
Calls: install_github ... request_fetch -> request_fetch.write_memory -> <Anonymous> -> .Call
Execution halted

```

I opened an R session with `/Library/Frameworks/R.framework/Versions/Current/Resources/bin/R` and attempted to install BiocCheck from GitHub : 

```
> library(devtools)
> install_github("Bioconductor/BiocCheck", tempdir())
```

This seemed to take a very long time, and so I killed it without letting it finish.  The output was : 

```
Downloading GitHub repo Bioconductor/BiocCheck@master
Installing BiocCheck
^C
```

The second time around ( I wanted to double-check something, before posting), I used the wrong "temp dir" (I forgot the 'e' in `tempdir()`) and it looks like I installed BiocCheck on morelia.  Although, I guess this wouldn't be in the same library that the SPB uses : 

```
> install_github("Bioconductor/BiocCheck", tmpdir())
Downloading GitHub repo Bioconductor/BiocCheck@master
Installing BiocCheck
^C
```

After restarting R : 

```
> library(BiocCheck)
> 
```

Unsure how to proceed, I sent another test package through the system.  So, I created https://tracker.bioconductor.org/issue1343 as a clone of https://tracker.bioconductor.org/issue3 .  

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;A side-note, I forgot to modify the DESCRIPTION file (so, "me@harvard.edu" might be getting some unwanted email, but hopefully that's not a real person).

After I kicked that off, it looks like we ran into a curl timeout again, but [only on Morelia](http://bioconductor.org/spb_reports/pkg1_0.0.1_buildreport_20151125123702.html#morelia_check_anchor).  The other nodes failed, but because the package just doesn't meet REQUIRED guidelines.

**Comment by b-long** [comment]
I've uploaded a new test package to the tracker under [issue1343](https://tracker.bioconductor.org/issue1343) .  It has built under job ID 1: http://staging.bioconductor.org:8000/jobs/1/ .  Tabling this for now, but will come back to it later on.

**Comment by b-long** [comment]
Overcome by events.
